qiita_pet/templates/study_description_templates/sample_template_tab.html

```diff
@@ -9,7 +9,7 @@
       {% end %}
     </select>
     <br/><br/>
-    <a class="btn btn-primary" onclick="process_sample_template();">Process sample template</a>
+    <a class="btn btn-primary" onclick="process_sample_template();">Process sample template</a> <a class="btn btn-primary" onclick="extend_sample_template();">Add samples to study</a>
```

Comment:
I can do that for all buttons in a separate pull request. Can you open an issue?